<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAQ_SystemClass</class>
 <widget class="QWidget" name="DAQ_SystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automatic Data Acquisition System</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mainLayout" stretch="1,2">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="leftImageLayout" stretch="0,0">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="subView1">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: black; border: 2px solid #4CAF50; border-radius: 10px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="subView2">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: black; border: 2px solid #4CAF50; border-radius: 10px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="mainView">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>600</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: black; border: 2px solid #4CAF50; border-radius: 10px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartUp">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: #4CAF50;
             color: white;
             border-radius: 10px;
             font-size: 18px;
             padding: 10px;
         }
         QPushButton:hover {
             background-color: #45a049;
         }
        </string>
       </property>
       <property name="text">
        <string>Startup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCapture">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: #4CAF50;
             color: white;
             border-radius: 10px;
             font-size: 18px;
             padding: 10px;
         }
         QPushButton:hover {
             background-color: #45a049;
         }
        </string>
       </property>
       <property name="text">
        <string>Capture Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShutdown">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
         QPushButton {
             background-color: #f44336;
             color: white;
             border-radius: 10px;
             font-size: 18px;
             padding: 10px;
         }
         QPushButton:hover {
             background-color: #d32f2f;
         }
        </string>
       </property>
       <property name="text">
        <string>Shutdown</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="DAQ_System.qrc"/>
 </resources>
 <connections/>
</ui>
